pkg/audit/ru.go

```diff
@@ -60,17 +64,31 @@ func NewRUCreationEventDetails(
 	return json.RawMessage(bytes), nil
 }
 
+type Labeler interface {
+	GetLabels(labelType labels.Type, id string) (labels.Labeled, error)
+}
+
 func NewRUCompletionEventDetails(
 	rollingUpdateID roll_fields.ID,
 	succeeded bool,
 	canceled bool,
+	labeler Labeler,
```

Comment:
i could also consider putting this function on the audit log store and making the audit log store require a labeler but felt it's slightly too early to that